src/pages/content-edit/versioning.js

```diff
@@ -28,6 +28,15 @@ module.exports = {
 
     view : function(ctrl, options) {
         return m("div", { class : options.class },
+            m("a", {
+                    title  : "Preview",
+                    href   : "/content/" + ctrl.schema + ctrl.id,
```

Comment:
I checked out the PR and ran it, that code creates this link: `http://localhost:9966/content/player-K7YakTr_1HdRPW6FZAg`

Which isn't valid. Doesn't matter though, since that link should be going out to the external preview URL anyways (assuming one is configured).